examples/clickhouse_cloud.rs

```diff
@@ -0,0 +1,88 @@
+use clickhouse::sql::Identifier;
+use clickhouse::Client;
+use clickhouse_derive::Row;
+use serde::{Deserialize, Serialize};
+use std::env;
+
+// This example requires three environment variables with your instance credentials to be set
+//
+// - CLICKHOUSE_URL (e.g., https://myservice.clickhouse.cloud:8443)
+// - CLICKHOUSE_USER
+// - CLICKHOUSE_PASSWORD
+//
+// Additionally, `rustls-tls` or `native-tls` cargo features should be enabled.
```

Comment:
Mm, it should be marked via `required_features` in Cargo.toml

Reply:
This is great. Added.